CanvasCore/CanvasCore/Helm/Helm.swift

```diff
@@ -369,6 +369,9 @@ open class HelmManager: NSObject {
                 callback?()
             } else {
                 viewController.addModalDismissButton(buttonTitle: nil)
+                if let disableSwipeDownToDismissModal: Bool = options[PropKeys.disableDismissOnSwipe] as? Bool, disableSwipeDownToDismissModal {
```

Comment:
I think this could be simplified
```suggestion
                if options[PropKeys.disableDismissOnSwipe] as? Bool == true {
```

Reply:
You are right, thanks!